src/logger.js

```diff
@@ -47,7 +47,7 @@ class Logger {
      * @param {string} errorCode (optional) machine readable error code (string in snake_case)
      * @param {string} payload (optional) JSON payload
      */
-  debug (message, payload = undefined, errorCode = undefined) {
+  debug (message, payload, errorCode) {
```

Comment:
These are not needed since all parameters are `undefined` in js by default

Reply:
OK